src/HotChocolate/Data/src/Data/Sorting/Convention/SortConvention.cs

```diff
@@ -126,8 +127,31 @@ protected internal override void Complete(IConventionContext context)
     }
 
     /// <inheritdoc />
-    public virtual string GetTypeName(Type runtimeType) =>
-        _namingConventions.GetTypeName(runtimeType, TypeKind.Object) + _typePostFix;
+    public virtual string GetTypeName(Type runtimeType)
+    {
+        var name = _namingConventions.GetTypeName(runtimeType, TypeKind.Object);
+
+        var isInputObjectType = runtimeType.IsAssignableTo(typeof(SortInputType));
```

Comment:
```suggestion
        var name = _namingConventions.GetTypeName(runtimeType);

        var isInputObjectType = typeof(SortInputType).IsAssignableFrom(runtimeType);
```

Let's try to keep this exactly the same as the `FilterConvention`.